notice_and_comment/templates/regulations/nc-homepage.html

```diff
@@ -0,0 +1,177 @@
+
+    <section class="good-comment">
+
+      <div class="good-comment-content">
+
+        <h2>Build a good comment:</h2>
+
+        <ul>
+          <li>
+            <div class="triangle"></div>
+            <span>
+              <strong>Explain why</strong> you agree or disagree; suggest alternatives and substitute language for your requested changes.
+            </span>
+          </li>
+          <li>
+            <div class="triangle"></div>
+            <span>
+              <strong>Provide specific examples</strong> to illustrate your concerns.
+            </span>
+          </li>
+          <li>
+            <div class="triangle"></div>
+            <span>
+              <strong>Base your comments on sound reasoning,</strong> scientific evidence, and/or how you will be impacted by the proposal.
+            </span>
+          </li>
+          <li>
+            <div class="triangle"></div>
+            <span>
+              <strong>Address trade-offs and opposing views.</strong>
+            </span>
+          </li>
+          <li>
+            <div class="triangle"></div>
+            <span>
+              <strong>Identify your relevant credentials and experience.</strong>
+            </span>
+          </li>
+          <li>
+            <div class="triangle"></div>
+            <span>
+              You don't have to comment on everything - you can <strong>select those issues that concern and affect you the most</strong> and/or you understand the best.
+            </span>
+          </li>
+        </ul>
+
+        <div class="more-tips">
+          Want more tips?
+        </div>
+
+        <div class="more-tips-links">
+          <a href="">EPA's tips for writing effective comments <span class="cf-icon cf-icon-external-link"></span></a><br/>
+          <a href="">Regulation.gov's detailed tips for commenting <span class="cf-icon cf-icon-external-link"></span></a>
+        </div>
+      </div>
+
+    </section>
+
+    <section class="about-this-tool">
+
+      <h2>About this tool</h2>
+
+      <div class="about-this-tool-content">
+
+        <img class="logo" src="{% static "regulations/img/about.png" %}" alt="About this tool"/>
+
+        <div class="about-this-tool-info">
+          <p>This eRegulations tool was developed by <a href="http://www.18f.gsa.gov">18F <span class="cf-icon cf-icon-external-link"></span></a> (part of the <a href="http://www.gsa.gov">General Services Administration <span class="cf-icon cf-icon-external-link"></span></a>) in partnership with EPA.</p>
+          <p><strong>Learn more about the eRegulations platform at <a href="http://eregs.github.io">eregs.github.io <span class="cf-icon cf-icon-external-link"></span></a></strong></p>
+        </div>
+
+      </div>
+
+      <div class="feedback">
+        <h4>We want to hear from you!</h4>
+        <p>To give feedback on this tool, <a href="">open a new issue in this repository <span class="cf-icon cf-icon-external-link"></span></a> (requires a free login).
```

Comment:
Would you mind opening an issue (or tacking on to another issue) to track this empty link?